### PR TITLE
Create Ui for user to interact with

### PR DESCRIPTION
**Pull request by AhmedMohsen600**
In this branch we:

- Created an app.rb file that does:
 - List all books.
 - List all people.
 - Create a person (teacher or student, not a plain Person).
 - Create a book.
 - Create a rental.
 - List all rentals for a given person's id.
 - Created main.rb define the entry point, this will be a method called main that is invoked at the end of your file. This method does the following:
 - Present the user with a list of options to perform.
 - Lets users choose an option.
 - If needed, ask for parameters for the option.
 - Have a way to quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+  def run
+    loop do
+      puts main_section
+      choice = get_number('', 'Please enter a valid number!')
+      return if choice == 7
+
+      run_available_options(choice)
+    end
+  end
```
Comment (TadesseDev) [inline app.rb:24]: - [ ] Dear @AhmedMohsen600, good job implementing the functionalities. As I notice there is a problem reading a student's name. After creating it, when I try to read the data it seems to replay `undefined`.

https://user-images.githubusercontent.com/69077061/216767173-e6347523-b986-4591-9620-115aa2b2bc65.mp4

- [x] Also, When I try to load `rentals`  by selecting a person id, there seems to be an exception. Please address this too🛑🛑.

Screenshot 2023-02-04 151203

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @AhmedMohsen600 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7

## Highlights
 - [ ]  Good job overall following the template design ✅
 - [ ]  Correct GitFlow  ✅
 - [ ]  Good commit history  ✅
 - [ ]  Good PR title/description  ✅
 - [ ]  All linters passed  ✅
 
## Required Changes ♻️
- [ ] Notice the link for your MIT file is broken, Please make sure to add the MIT license file and update its content in regard to your name and project. Also replace the `[YOUR NAME]` part with your name, and `[APP TYPE]` with your app type.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — COMMENTED** [review]
Hello @AhmedMohsen600  👋,

_This is an additional review as a result of the second opinion request._

- I have been able to check your project and the requested changes have been implemented, Optionally, I would suggest you implement the `List all rentals for a given person` functionality as it is throwing an error.


https://user-images.githubusercontent.com/66112760/216776823-73711e8c-42f9-43ec-9b51-0d3348331fb2.mov


As this was the only reason that your project was not approved it will be approved as soon as possible.
Please note that since we will approve it manually so this could take a few hours!

Keep rocking 🚀!

**Comment by AhmedMohsen600** [comment]
@bimbolabuari I have fixed `List all rentals for a given person` issue and pushed it

**Comment by bimbolabuari** [comment]
@AhmedMohsen600, nice job 👍 
You will get approved manually no worries 🙂

**Pull request merged**